chainer/functions/array/get_item.py

```diff
@@ -37,12 +54,19 @@ def get_item(x, slices):
 
     Args:
         x (tuple of Variables): Variable to be sliced.
-        slices (slice or tuple of slices): Slice objects to slice variable.
+        slices (int, slice, None or Ellipsis or tuple of them): Basic slicing
+            to slice a variable. It supports ``int``, ``slice``, ``newaxis``
+            (equivalent to ``None``) and ``Ellipsis``.
 
     Returns:
         Variable: :class:`~chainer.Variable` object
             which contains sliced array of ``x``.
 
+    .. note::
+
+       See NumPy documents for details of `indexing
+       <http://docs.scipy.org/doc/numpy/reference/arrays.indexing.html>`_
```

Comment:
(Very minor comment) Please end the sentence with period(.). Other parts looks good to me.